src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java

```diff
@@ -112,7 +112,9 @@ public void execute_duplicateOrderUnfilteredList_warningGiven() {
         EditOrderDescriptor descriptor = new EditOrderDescriptorBuilder(firstOrder).build();
         EditOrderCommand editOrderCommand = new EditOrderCommand(INDEX_THIRD_PERSON, descriptor);
 
-        String expectedMessage = EditOrderCommand.MESSAGE_DUPLICATE_ORDER_WARNING
+        String warningMessage = String.format(EditOrderCommand.MESSAGE_DUPLICATE_ORDER_WARNING,
+                firstOrder.getStatus().toString().toLowerCase());
```

Comment:
![image](https://github.com/user-attachments/assets/84b20757-589a-4921-9be5-8257a8f37040)
kx made this method is previous commit.

Reply:
changed